### PR TITLE
[REF] payment_term_type: Added a Redme and  compleate help field descriptions

### PR DESCRIPTION
**Pull request by hugho-ad**
FIX https://github.com/Vauxoo/addons-vauxoo/issues/908

DUMMY https://github.com/Vauxoo/yoytec/pull/2124

### CODE REVIEW

**Comment by hugho-ad** [comment]
@zaoral 

Here is the documentation

File: payment_term_type/README.rst
Comment (zaoral) [inline payment_term_type/README.rst]: remove the \n

File: payment_term_type/README.rst
Comment (zaoral) [inline payment_term_type/README.rst]: When you say

> In Settings> Account you can find a selection field payment_type 

I think will better to refer the exact path of the menu and mention the human readable name of the field, for example: 

> In the`Settings > Configuration > Accounting` wizard you can find the the **Payment Term Type** (`payment_type)` option

File: payment_term_type/README.rst
Comment (zaoral) [inline payment_term_type/README.rst:24]: > The payment term will be cash type **when the payments have been covered in
>  just one exhibition.** without matter the date of payment or **when the payments
>  have been covered in just one exhibition**, in the same day that the  sale
>  order be confirmed.

This paragraph repeat the same sentence twice in he form this or that, please could you rewrote this paragh

File: payment_term_type/README.rst
Comment (zaoral) [inline payment_term_type/README.rst]: > `here <https://github.com/Vauxoo/addons-vauxoo/issues/new?body=module:%20payment_term_type%0Aversion:%201.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**&title=[8.0]%20yoytec_rma_workflow:%20problem%20summary%20here>`_.

You are referring to yoytec_rma_workflow, please update to refer to the current module. 
image

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @hugho-ad 

Thank you for add the modules README.rst.
I leave you some changes before merge.

Best Regards

Please could you

**Comment by hugho-ad** [comment]
@zaoral thanks for your reviewing, Comments Done

**Comment by hugho-ad** [comment]
@zaoral can you merge if you agree?

**Review by zaoral — APPROVED** [review]
Yes
I will perform the merge
Thank you for the contribution @hugho-ad

**Comment by zaoral** [comment]
I have not merge permissions xD.

@moylop260 
Please can you help us with this merge?

Best Regards

**Pull request merged**
